main.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Just For Today</title>
+</head>
+<body>
+    <div id="clock">
+        <div id="hours"></div>
+        <div id="minutes"></div>
+        <div id="seconds"></div>
+    </div>
+
+    <script src="main.js"></script>
+</body>
+</html>
```

Comment:
<img width="449" alt="Screen Shot 2023-01-04 at 7 34 19" src="https://user-images.githubusercontent.com/60898706/210490707-ce7fa587-6f5a-4fc7-9fc7-aaec68ad7dec.png">

Looks good! Just add an empty line to the end of the file, per this recommendation: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Do you have any sort of linter (like eslint) installed in vscode? And/or Prettier? We can review these things if you would like!

Reply:
I have eslint...

Reply:
Ok, cool, I'll look into this more. Thanks!